### PR TITLE
[YUNIKORN-2461] Consider supporting /ws/v1/partition/{partitionName}/queue/{queueName}/applications/:state

### PR DESCRIPTION
**Pull request by blueBlue0102**
### What is this PR for?
Add documentation for the API endpoint `/ws/v1/partition/{partitionName}/queue/{queueName}/applications/:state`.

### What type of PR is it?
* [x] - Documentation

### What is the Jira issue?
[YUNIKORN-2461](https://issues.apache.org/jira/browse/YUNIKORN-2461)

### Screenshots (if appropriate)
image

### CODE REVIEW

File: docs/api/scheduler.md [hunk 1]
```diff
@@ -646,6 +646,31 @@ In the example below there are three allocations belonging to two applications,
 
 **Code** : `500 Internal Server Error`
 
+### Queue applications by state
+
+Fetch all Applications for the given Partition/Queue/State combination and displays general information about the applications like used resources, queue name, submission time and allocations. Only following application states are allowed: active, rejected, completed. Active is a fake state that represents all application states except completed and rejected. For active state, can narrow the result by status query parameters(case-insensitive). For example, can fetch `Running` applications for the default partition and root queue by `/ws/v1/partition/default/queue/root/applications/active?status=running`. In case the queue name contains any special characters, it needs to be url escaped to avoid issues.
```
Comment (chia7712) [inline docs/api/scheduler.md:651]: > Only following application states are allowed: active, rejected, completed.

`rejected` and `completed` are not acceptable, right?

**Review by chia7712 — COMMENTED** [review]
@blueBlue0102 thanks for this patch!

File: docs/api/scheduler.md (hunk 1 above)
Comment (blueBlue0102) [reply docs/api/scheduler.md]: Sorry about the mistake. The error has been corrected, and I've made some changes for more clarity regarding the active state.

**Review by chia7712 — APPROVED** [review]
LGTM

**Pull request closed without merging**